package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "atom-shell",
-  "version": "1.2.1",
+  "version": "2.0.0",
```

Comment:
Don't bump the version in other peoples projects. It should be up to them what to release it as and when.